main.go

```diff
@@ -59,21 +58,23 @@ func main() {
 
 	resolver := depresolver.NewDependencyResolver()
 	config, err := resolver.ResolveOperatorConfig()
-	// LoggerFactory creates logger ALWAYS - no matter what isn't resolved
-	logger := controllers.NewLogger(config).Get()
+	// Init creates logger ALWAYS - no matter what isn't resolved
```

Comment:
Could you rephrase this comment? not really clear

Reply:
Hi @ytsarev,  amended. 

Regarding this concrete comment: 
We are creating logger from configuration, but if resolving configuration fails, from what we create logger to print error? The solution is to initialize default logger in case of configuration failed. 

I change comment to:
_Initialize desired logger or default logger in case of configuration failed_